CRM/Upgrade/Incremental/MessageTemplates.php

```diff
@@ -201,10 +201,11 @@ protected function getTemplateUpdates() {
       ],
       [
         'version' => '5.23.alpha1',
-        'upgrade_descriptor' => ts('Add Contributor Name to Offline Contribution receipts'),
+        'upgrade_descriptor' => ts('Add Contributor Name to Offline Contribution receipts; fix bad event self-service URL'),
         'templates' => [
           ['name' => 'contribution_offline_receipt', 'type' => 'text'],
           ['name' => 'contribution_offline_receipt', 'type' => 'html'],
+          ['name' => 'participant_confirm_html', 'type' => 'html'],
```

Comment:
@MegaphoneJon this should be

` ['name' => 'participant_confirm', 'type' => 'html'],`

instead of 
` ['name' => 'participant_confirm_html', 'type' => 'html'],`

Reply:
@yashodha Are you sure? Wouldn't that make civi look for `xml/templates/message_templates/participant_confirm_html_html.tpl` ?

```
  public function getTemplatesToUpdate() {
    $templates = $this->getTemplateUpdates();
    $return = [];
    foreach ($templates as $templateArray) {
      if ($templateArray['version'] === $this->getUpgradeVersion()) {
        foreach ($templateArray['templates'] as $template) {
          $return[$template['name'] . '_' . $template['type']] = array_merge($template, $templateArray);
        }
      }
    }
    return $return;
  }
```